src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/base/GT_MetaTileEntity_MultiblockBase_EM.java

```diff
@@ -195,6 +195,8 @@ public abstract class GT_MetaTileEntity_MultiblockBase_EM
     // if u need to force some things to be fixed - u might need to override doRandomMaintenanceDamage
     protected byte minRepairStatus = 3;
 
+    public boolean isNeededMaintenanceChecks = true;
```

Comment:
The variable name is hard to read imo. Consider something like `hasMaintenanceChecks`, `doesMaintanenceChecks`, ...